stl/inc/xutility

```diff
@@ -2507,16 +2507,19 @@ namespace ranges {
     public:
         using _Not_quite_object::_Not_quite_object;
 
-        template <class _It, sentinel_for<decay_t<_It>> _Se>
-        _NODISCARD constexpr iter_difference_t<decay_t<_It>> operator()(_It&& _Raw_first, _Se _Last) const
-            noexcept(_Nothrow_dist<_It, _Se>) /* strengthened */ {
-            if constexpr (sized_sentinel_for<_Se, decay_t<_It>>) {
-                return _Last - static_cast<const decay_t<_It>&>(_Raw_first); // Per LWG-3664
-            } else {
-                auto _First = _STD forward<_It>(_Raw_first);
-                _Adl_verify_range(_First, _Last);
-                return _Distance_unchecked(_Get_unwrapped(_STD move(_First)), _Get_unwrapped(_STD move(_Last)));
-            }
+        template <class _It, sentinel_for<_It> _Se>
+            requires(!sized_sentinel_for<_Se, _It>)
+        _NODISCARD constexpr iter_difference_t<_It> operator()(_It _First, _Se _Last) const
+            noexcept(noexcept(_Distance_unchecked(

```

Comment:
Until `_Get_unwrapped` is `noexcept`, this will not be noexcept, and MSVC with `/permissive` thinks `_Get_unwrapped` is noexcept for some godforsaken reason (this is why the test is failing)

Reply:
(this should probably be an issue - making `_Get_unwrapped` noexcept)